hooks/useUserAttribute.ts

```diff
@@ -0,0 +1,42 @@
+import { useContractRead } from 'wagmi'
+import kamonNFTContract from '@/utils/abis/kamonNFT.json'
+import { ethers } from 'ethers'
+
+type UserAttribute = {
+  point: number
+  claimableToken: number
+  answeredAt: string
+}
+
+export const useUserAttribute = (
+  contract: string,
+  owner: string | undefined
+): { data?: UserAttribute; isError: boolean; error: Error | null } => {
```

Comment:
I need to debug because I wasn't getting good data locally.

Reply:
https://github.com/henkaku-center/omise-interface/pull/62/commits/5342f195d938d3044813c3f70a130f47522d8ea4 fixed